### DISCUSSION TITLE
Vertical Taskbar for Windows 11 - huge tray icon area

### DISCUSSION THREAD

**Discussion by WegnerDan** [ramensoftware/windhawk-mods#1067] [General]
I really like that I can finally have a vertical taskbar in win11! 
But the tray icon area is really large. On Win10 and before, I used to have a similar taskbar setup, but the icons weren't in one row, but took up the available horizontal and vertical space. 

Is there a way to get the same thing with this mod? 

![image](https://github.com/user-attachments/assets/3426c80c-6b93-46df-b855-da577a93719f)

I know I have a lot of crap running, but I'm using all of it. 
I've already disabled the widgets, so the weather stuff isn't displayed anymore. 

I made photoshop mockup of what I mean:
![taskbar_modified_tray](https://github.com/user-attachments/assets/7cd7a220-1cb9-4920-ab9d-0b0464d9f71a)

**Comment by m417z** [comment]
Hi @WegnerDan, thanks for the feedback. This comes up quite often and is worth tracking. I created an issue for it: https://github.com/ramensoftware/windhawk-mods/issues/1068.
See my reply here: https://github.com/ramensoftware/windhawk-mods/issues/1068#issuecomment-2408557523.
